guides/doc-Content_Management_Guide/topics/Managing_Content_Views.adoc

```diff
@@ -425,7 +425,7 @@ In {Project}, you can use the package dependency resolution feature to ensure th
 You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
 
 Note that resolving package dependencies can cause significant delays to Content View promotion.
-The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.
+The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View so `{package-update-project}` is not guaranteed to work.
```

Comment:
This is good.  Do you think it would make sense to recommend particular workflows that are known to have less issues?  For instance, only include filters get much benefit from dependency solving, exclude filters should generally work fine without it.

Reply:
@sjha4 Can you comment?

Reply:
I saw that, but I was just thinking that even within the examples we give, some are definitely more foolproof than others.  For instance we don't clarify that if exclude-only filters don't need depsolving 99% of the time, everything becomes much much much faster to process.

Reply:
How should the wording be changed to meet your requirements?